src/components/PostDetails.tsx

```diff
@@ -1,117 +1,130 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from './Loader';
+
+import { Comment } from '../types/Comment';
+import { Post } from '../types/Post';
+import { getComment, deleteComment } from '../api/ApiMethods';
 import { NewCommentForm } from './NewCommentForm';
 
-export const PostDetails: React.FC = () => {
+type Props = {
+  isPostOpen: Post | null,
```

Comment:
```suggestion
  openedPost: Post | null,
```
we usually name var with `is` if it's boolean

Reply:
how can it be null if we check if this prop exists
<img width="307" alt="image" src="https://github.com/mate-academy/react_dynamic-list-of-posts/assets/70220667/4461cfa0-d017-4f30-86c7-3e25382df360">